packages/channels/src/teams/config.ts

```diff
@@ -0,0 +1,12 @@
+import Joi from 'joi'
+import { ChannelConfig } from '../base/config'
+
+export interface TeamsConfig extends ChannelConfig {
+  appId: string
+  appPassword: string
+}
```

Comment:
Removed the tenantId here because I don't know what it's used for. We can add it back if we need it.

Reply:
I think it was for some validation but I'm not sure either.